app.go

```diff
@@ -135,6 +135,13 @@ func (s *App) Stop(ctx context.Context) error {
 	return withTimeout(ctx, s.lifecycle.stop)
 }
 
+// Done allows blocking on SIGINT or SIGTERM
+func (App) Done() <-chan os.Signal {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
+	return c
+}
```

Comment:
Part of the goal here is, I think, to hide away from application developers what the source of the stop signal is. If we expose a channel of `os.Signal`, we lose some of that benefit. We also lose the possibility of making the app satisfy the `context.Context` interface.